.github/workflows/CI.yml

```diff
@@ -225,6 +225,8 @@ jobs:
       - name: PIP install required packages
         shell: bash -l {0}
         run: |
+            # Package lpynn has lpython_emulation as dependency
+            # Hence, it should by default install lpython_emulation
             python -m pip install lpynn numpy
```

Comment:
I think that this is problematic. If we change anything in `lpython.py` this test will break, since we have to update it in `lpynn` as well.

At the very least, let's pin a specific version of the lpynn package, as well as numpy.

Reply:
Done.